composer.json

```diff
@@ -22,29 +22,29 @@
     "prefer-stable": true,
     "require": {
         "php": ">= 7.2",
-        "linio/input": "^2.5",
+        "linio/input": "^2.7",
         "psr/container": "~1.0",
         "psr/http-message": "^1.0",
-        "psr/log": "~1.0",
-        "ramsey/uuid": "^3.8",
+        "psr/log": "~1.1",
+        "ramsey/uuid": "^3.9|^4.0",
```

Comment:
Is 3.9 necessary?

Reply:
Just a precaution, we have not yet updated our APIs, I do not know what scope this change would have.

Reply:
I see, thanks!

Reply:
Remember this library is public.
If the `4.x` version brings BC to this library we should just do a new release here bumping the major version.

Reply:
I think if it would bring some BC.

https://uuid.ramsey.dev/en/latest/upgrading/3-to-4.html

Reply:
I think what you did is ok! 😄